tests/ci/docker_images_check.py

```diff
@@ -173,6 +173,11 @@ def parse_args() -> argparse.Namespace:
 
 
 def main():
+    NAME = "Push to Dockerhub " + (
+        "aarch64"
+        if any(arch in platform.processor() for arch in ("aarch", "arm"))
+        else "amd64"
+    )
```

Comment:
we can simplify it
```suggestion
    NAME = f"Push to Dockerhub {platform.machine()}"
```

Much shorter, and good enough

Reply:
we have aarch64 and amd64 in all other similar cases

Reply:
No problem, let's use `args.suffix` without any other logic

Reply:
`parser.add_argument("--suffix", type=str, required=True, help="arch suffix")` is already there

Reply:
what is the problem with current implementation? I don't think we need input parameter to define job name

Reply:
> we have aarch64 and amd64 in all other similar cases
with this I meant that processor.machine() gives suffixes different from what we use in ci. that's why there is that additional if/else to map it into aarch64/amd64

Reply:
The problem it doesn't use the info we have and adds the unnecessary differences between https://github.com/ClickHouse/ClickHouse/blob/master/.github/workflows/reusable_docker.yml#L42 + https://github.com/ClickHouse/ClickHouse/blob/master/.github/workflows/reusable_docker.yml#L27

We have it already. Let's use it